dissect/target/plugins/os/unix/linux/packagemanager.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
```

Comment:
The Packagemanager namespace plugin was put under the unix subdir on purpose. Even though there are no plugins yet for other OSes than Linux.